aten/src/ATen/native/native_functions.yaml

```
@@ -8350,6 +8350,15 @@
     CPU: _convert_indices_from_coo_to_csr_structured_cpu
     CUDA: _convert_indices_from_coo_to_csr_structured_cuda
 
+- func: _convert_indices_from_csr_to_coo(Tensor crow_indices, Tensor col_indices, int size, *, bool out_int32=False) -> Tensor
```

There is no need to specify the `size` parameter as it is equal to `col_indices.numel()`. Hence:
```suggestion
- func: _convert_indices_from_csr_to_coo(Tensor crow_indices, Tensor col_indices, *, bool out_int32=False) -> Tensor
```
Of course, this suggestion applies to all of the PR change-set that must be updated accordingly.